routes.js

```diff
@@ -1,12 +1,22 @@
 const dashboard = require('./dashboard')
 
 function configureRoutes(app) {
+  app.get('/', dashboard.sessionChecker, dashboard.redirectToHomePage)
   app.get('/clients/new', dashboard.sessionChecker, dashboard.renderNewClientPage) // Must be configured before /clients/:id
   app.get('/clients/:id', dashboard.sessionChecker, dashboard.renderClientDetailsPage)
   app.get('/clients/:id/edit', dashboard.sessionChecker, dashboard.renderClientEditPage)
+  app.get('/dashboard', dashboard.sessionChecker, dashboard.renderDashboardPage)
+  app.get('/locations/new', dashboard.sessionChecker, dashboard.renderNewLocationPage)
+  app.get('/locations/:locationId', dashboard.sessionChecker, dashboard.renderLocationDetailsPage)
+  app.get('/locations/:locationId/edit', dashboard.sessionChecker, dashboard.renderLocationEditPage)
+  app.get('/login', dashboard.renderLoginPage)
+  app.get('/logout', dashboard.submitLogout)
 
   app.post('/clients', dashboard.validateNewClient, dashboard.submitNewClient)
   app.post('/clients/:id', dashboard.validateEditClient, dashboard.submitEditClient)
+  app.post('/locations', dashboard.validateNewLocation, dashboard.submitNewLocation)
+  app.post('/locations/:locationId', dashboard.validateEditLocation, dashboard.submitEditLocation)
+  app.post('/login', dashboard.submitLogin)
```

Comment:
so, in your vision for this, would we add the sensorEvent and xethru routes to here, and call functions in xethru.js and sensorEvent.js files which would host the handler code + helpers?

Reply:
Yes, exactly like that. Does that make sense to you too?

An other option would be to not have this centralization and instead just have all these dashboard-related routes at the top of `dashboard.js` and then have sensor routes at the top of `sensors.js` (or whatever that split ends up looking like).